seatunnel-connectors-v2/connector-doris/src/test/java/org/apache/seatunnel/connectors/doris/rest/models/RestServiceTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.doris.rest.models;
+
+import org.apache.seatunnel.connectors.doris.rest.RestService;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Slf4j
+public class RestServiceTest {
+
+    @Test
+    void testRandomEndpoint() {
+
+        List<String> list = Arrays.asList("fe_host1:fe_http_port1", "fe_host2:fe_http_port2");
+        Assertions.assertTrue(
+                list.contains(RestService.randomEndpoint("fe_host1:fe_http_port1", log)));
+        Assertions.assertTrue(
+                list.contains(
+                        RestService.randomEndpoint(
+                                "fe_host1:fe_http_port1,fe_host2:fe_http_port2", log)));
+        Assertions.assertTrue(
+                list.contains(
+                        RestService.randomEndpoint(
+                                "fe_host2:fe_http_port2,fe_host1:fe_http_port1", log)));
```

Comment:
This test case can not prove the endpoint is random. Please refer https://github.com/apache/seatunnel/blob/df1954d52076c7ba6d3d764c9e4e58725193a102/seatunnel-engine/seatunnel-engine-server/src/test/java/org/apache/seatunnel/engine/server/resourcemanager/ResourceManagerTest.java#L86

Reply:
done